### DISCUSSION TITLE
Not getting Expected Excel

### DISCUSSION THREAD

**Discussion by vishal-chaudhary-here** [jxlsteam/jxls#216] [Q&A] [answered]
I tried following example. But I am not getting expected result.

Template:
[employee_group_by.xlsx](https://github.com/jxlsteam/jxls/files/9622122/employee_group_by.xlsx)

Code:
`
import java.io.FileOutputStream;
import java.io.IOException;
import java.io.InputStream;
import java.io.OutputStream;
import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;
import java.util.Locale;

import org.jxls.common.Context;
import org.jxls.util.JxlsHelper;

import com.mo.one.jxls.entity.Employee;

public class Test {
	
	public static List<Employee> generateSampleEmployeeData() throws ParseException {
	    List<Employee> employees = new ArrayList<>();
	    SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MMM-dd", Locale.US);
	    employees.add( new Employee("Elsa", dateFormat.parse("1970-Jul-10"), 1500, 0.15) );
	    employees.add( new Employee("John", dateFormat.parse("1975-Oct-05"), 2500, 0.00) );
	    return employees;
	}
	
	public void generateExcel() throws IOException, ParseException {
		try (InputStream is = Test.class.getClassLoader().getResourceAsStream("employee_group_by.xlsx")) {
			try (OutputStream os = new FileOutputStream("target/employee_group_by.xlsx")) {
				JxlsHelper.getInstance().processTemplate(is, os, getContext());
			}
		}
	}
	
	private Context getContext() throws ParseException {		
		Context context = new Context();
		context.putVar("employees", generateSampleEmployeeData());
		context.putVar("columnNames", Arrays.asList("Payment", "Bonus"));
		return context;
	}
}
`

Actual Output (With extra lines):
![image](https://user-images.githubusercontent.com/20570277/191656501-d1eae7b9-fa78-402f-b70b-6a2a7a3b67a6.png)

Expected Output (Without extra lines):
![image](https://user-images.githubusercontent.com/20570277/191656581-f4b9522b-0a17-43f0-8bf7-825c771e1cf1.png)

Please have a look. Thanks in advance .

**Comment by SoltauFintel** [comment, marked as answer]
My guess after a quick look at the template: lastCell of cell A3 must not be A3 but B4 ??

**Reply by vishal-chaudhary-here** [reply]
Hi, I tried but this is giving result like:
![image](https://user-images.githubusercontent.com/20570277/193511893-8cc1af6d-3680-4216-afc6-cb6a8cff1c5f.png)

Thanks
